src/Components/Common/ConfirmDialogV2.tsx

```diff
@@ -0,0 +1,55 @@
+import DialogModal from "./Dialog";
+import ButtonV2, { ButtonVariant } from "./components/ButtonV2";
+
+type ConfirmDialogV2Props = {
+  title: React.ReactNode;
+  description?: React.ReactNode;
+  disabled?: boolean;
+  show: boolean;
+  action: string;
+  variant?: ButtonVariant;
+  onClose: () => void;
+  onConfirm: () => void;
+  children?: React.ReactNode;
+  cancelText?: React.ReactNode;
+};
+
+const ConfirmDialogV2 = (props: ConfirmDialogV2Props) => {
+  const {
+    title,
+    description,
+    show,
+    disabled,
+    variant,
+    action,
+    onClose,
+    onConfirm,
+    cancelText,
+    children,
+  } = props;
+
+  return (
+    <DialogModal
+      onClose={onClose}
+      title={title}
+      description={description}
+      show={show}
+    >
+      {children}
+      <div className="mt-4 flex justify-between">
+        <ButtonV2 onClick={onClose} variant="danger">
+          {cancelText || "Cancel"}
+        </ButtonV2>
```

Comment:
Why would cancel operation be danger variant?
```suggestion
        <ButtonV2 onClick={onClose} variant="secondary">
          {cancelText || "Cancel"}
        </ButtonV2>
```